install-cni.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e -x;
+
+if [ -w "/host/opt/cni/bin/" ]; then
+    cp -f /install/bin/* /host/opt/cni/bin/;
```

Comment:
we are coping executable file  from this path to host path. Please make sure we are making it executable before copying.use chmod to make it executable.

Reply:
Done